src/ep.config.json

```diff
@@ -27,12 +25,12 @@
     }
   ],
   "defaultCurrencyValue": "CAD",
-  "skuImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%sku%.png",
-  "siteImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%fileName%",
+  "skuImagesUrl": "https://refstoreimgs.s3-us-west-2.amazonaws.com/productImages/%fileName%",
+  "siteImagesUrl": "https://refstoreimgs.s3-us-west-2.amazonaws.com/siteImages/%fileName%",
```

Comment:
Getting CORS errors on these images. Need to add a CORS config for the new bucket:
![image](https://user-images.githubusercontent.com/39800563/81725517-f3b2d000-9453-11ea-93d7-3ffc00bc8c1c.png)

Reply:
Will place the new images in the demo-team AWS

Reply:
Updated the url

Reply:
FYI I get cors issues even when switching back to elasticpath-demo-images